docs/sdkx-react-native/delegates.mdx

```diff
@@ -0,0 +1,280 @@
+---
+sidebar_position: 100
+title: Helpshift Delegates
+description: "The Helpshift SDK provides delegate callbacks to help app developers track a user's activities within the help section."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# Helpshift Delegates {#delegates}
+
+<Intro>
+
+The Helpshift SDK provides delegate callbacks to help app developers track a user's activities within the help section.
+
+</Intro>
+
+## Helpshift Delegate implementation {#delegate-implementation}
+
+- To use Helpshift Delegate please import `helpshiftEmitter`
+For example:
+```javascript
+import {
+  helpshiftEmitter,
+} from 'helpshift-plugin-sdkx-react-native';
+
+```
+This event contains information about the Message count and how to handle the data based on the platform.
```

Comment:
Dont need this statement here. It does not have any header to indicate this is for receivedUnreadMessageCount.
This is just an example code to show the import and usage of helpshiftEmitter.
Just add a statement that the following is an example code for how to use helpshiftEmitter